qlog/src/reader.rs

```diff
@@ -24,9 +24,15 @@
 // NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 // SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-use crate::events::Event;
 use crate::QlogSeq;
 
+/// Represents the format of the read event, either standard qlog::events::Event
```

Comment:
Seems better to remove the "either ..." part, and add something to each enum case instead.